src/frontend/src/pages/PPQPage/PPQPage.scss

```diff
@@ -1,33 +1,34 @@
-.ppq-container {
+.get-started-container {
```

Comment:
We should NOT be defining this like this. There is nothing unique about this style for this container.

We should be thinking about defining classes that apply to the whole application. Not to individual boxes.

Reply:
If the box is unique we need to have a conversation with UI/UX. This box is not unique.

Reply:
yes, I just replace this with bcgovPageContainer

Reply:
let me know if it works in your local, looks good in my local